app/controllers/casa_admins_controller.rb

```diff
@@ -135,7 +135,7 @@ def set_admin
   end
 
   def update_casa_admin_params
-    CasaAdminParameters.new(params).with_only(:email, :display_name, :phone_number, :monthly_learning_hours_report)
+    CasaAdminParameters.new(params).with_only(:email, :display_name, :phone_number, :date_of_birth, :monthly_learning_hours_report)
```

Comment:
Good job adding date of birth to the accepted params to meet the requirements of PR.